fieldsolver/derivatives.cpp

```diff
@@ -766,19 +777,21 @@ void calculateScaledDeltas(
    std::array<Real, 3> myJ = {dBZdy - dBYdz, dBXdz - dBZdx, dBYdx - dBXdy};
    Real BdotJ {0.0};
    Real Bsq {0.0};
+   Real J {0.0};
    for (int i = 0; i < 3; ++i) {
       BdotJ += myB[i] * myJ[i];
       Bsq += myB[i] * myB[i];
+      J += myJ[i] * myJ[i];
    }
+   J = std::sqrt(J);
```

Comment:
Is this *really* certain never to be 0 by "chance", "bad luck" or "design", e.g. at t=0?

Reply:
No, but sqrt(0) would just be 0 no?

Reply:
Might it be `sqrt(-0)`?

Reply:
Shouldn't be negative. I'm going to be upset if IEEE floats allow a sum of square numbers to be...